Cargo.toml

```diff
@@ -665,6 +666,7 @@ sinks-splunk_hec = []
 sinks-statsd = ["sinks-utils-udp", "tokio-util/net"]
 sinks-utils-udp = []
 sinks-vector = ["sinks-utils-udp", "tonic", "protobuf-build"]
+sinks-websocket = ["tokio-tungstenite"]
```

Comment:
`sinks-websocket` also needs to be added to the `sinks-logs` list above, which will add it as a default feature